django/mgmt/static/js/downsample.js

```diff
@@ -118,6 +118,10 @@ function downsample_ajax(collection, experiment, channel, type, data){
             500: function (response) {
                 raise_ajax_error(response);
                 $("#downsample-btn").removeClass('disabled');
+            },
+            409: function (response){
+                raise_ajax_error(response);
+                $("#downsample-btn").removeClass('disabled');
```

Comment:
this is currently in #52 I think — maybe we don't need both? feel free to close mine if you prefer

Reply:
Oops sorry completely forgot about #52 . I'll delete mine since I already approved yours